config/gulp/sass.js

```diff
@@ -33,6 +36,43 @@ gulp.task('scss-lint', function (done) {
 
 });
 
+gulp.task('create-tmp-docs', function (done) {
+  return gulp.src('docs/doc_assets/css/**/*.scss')
+    .pipe(replace(/---/g, ''))
+    .pipe(gulp.dest('tmpdocs'));
+});
+
+gulp.task('lint-tmp-docs-scss', function (done) {
```

Comment:
When this lint tasks finds errors, will it point to this `tmpdocs` directory as the path of the files with problems?